tests/integration_tests/functional/test_logging.py

```diff
@@ -58,9 +58,9 @@ def check_log_message_format(log_str, instance_id, level, show_level, show_origi
         pattern += ":(" + "|".join(LOG_LEVELS) + ")"
     if show_origin:
         pattern += ":([^:]+/[^:]+):([0-9]+)"
-    pattern += "\\]"
```

Comment:
Nit: This is not needed right? as `match` later matches from the beginning, but not to the end.

No need to change.